config/crd/bases/neutron.openstack.org_neutronapis.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.10.0
+    controller-gen.kubebuilder.io/version: v0.9.2
```

Comment:
Just curious, what's the reason behind this?

Reply:
This got updated after pre-commit run.

Reply:
can you try again after dropping rm bin/controller-gen from local neutron-operator clone. With that these files will not change, v0.10.0 is the current version as per CONTROLLER_TOOLS_VERSION ?= v0.10.0 in Makefile

Reply:
Well this works.